website/docs/guides/migration_guide_alert_conditions.html.markdown

```diff
@@ -182,7 +194,9 @@ resource "newrelic_nrql_alert_condition" "foo" {
 }
 ```
 
-### newrelic_infra_alert_condition: Process not running
+#### newrelic_infra_alert_condition: Process Not Running
+
+The following example illustrates changing an infra alert condition for 'Process Not Running' to an NRQL-based alert condition.
 
 Example newrelic_infra_alert_condition:
```

Comment:
@vagrawal-newrelic can you please add backticks to all references of such resources we have (above all code snippets we have in this page, synthetics and infra), similar to the one I've referenced in this comment

Reply:
Added Backtick for all references in migration guide for infra and synthetic.